Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -19,9 +47,26 @@ public static void Sort(int[] array)
         /// <param name="array">Input array in a random order.</param>
         public static void Reverse(int[] array)
         {
-            // ToDo: implement.
-        }
+            var stop = 0; 
 
+            if (array == null)
+            {
+                stop = -1; 
+            }
+            else
+            {
+                stop = array.Length / 2;
+            }
+             
+                      
+            for (var i = 0; i < stop; i++)
+			{
+                var temp = array[i]; 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['array' is null on at least one execution path.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887669)